single-node/user-data

```diff
@@ -208,7 +208,7 @@ WantedBy=multi-user.target
 EOF
     fi
 
-    local TEMPLATE=/etc/kubernetes/manifests/kube-proxy.yaml
+    local TEMPLATE=/etc/kubernetes/addons/kube-proxy.yaml
```

Comment:
I missed that the kubelet `--config` directory was also changed - and didn't see how this would work. I'll comment inline with that change.